### ISSUE TITLE
Image: direct support for alternate text

### ISSUE THREAD

**Issue by bhainesva** [yext/pages#155]
### Description

All images should have alt text (or explicitly set an empty alt text attribute). Currently this is possible using the `imgOverrides` prop, but I think alt text is important and frequently used enough that it deserves top level support.

### Suggested solution

In the Image component implementation we could provide an explicit `alt` prop. 

### Validations

- [X] Check that there isn't already an issue that requests the same feature to avoid creating a duplicate.

**Comment by mkilpatrick** [comment]
Fixed in https://github.com/yext/pages/pull/180
